.github/workflows/create-tag-from-mix-version.yaml

```diff
@@ -0,0 +1,66 @@
+name: create-tag-from-mix-version
+on:
+  workflow_call:
```

Comment:
Is the assumption that each project's CI workflow will invoke this workflow when building a commit on the `main` branch?

Reply:
Yeah. It should look something like this:
```yaml
name: publish
on:
  pull_request:
    types: [closed]
jobs:
  create-tags-from-mix-file:
    name: Create Tags from mix file
    if: github.event.pull_request.merged == true
    uses: rentpath/actions-services-workflows/.github/workflows/create-tag-from-mix-version.yml#d37afcc
    secrets: inherit
```